### ISSUE TITLE
how to use for blog

### ISSUE THREAD

**Issue by uurcank** [sillsdev/docu-notion#41]
Is it possible to use docu-notion to manage blog articles in docusaurus?

**Comment by hatton** [comment]
Sorry, docu-notion doesn't know about the blog part of Docusaurus at this time.

**Comment by uurcank** [comment]
@hatton in theory we can add more columns to notion database and use it to generate markdown. It is no different than docs.

what would a plugin for this look like? plugin dev documentation does not explain anything. can you at least point how to generate a blog folder with a plugin?

**Comment by hatton** [comment]
Right, I don't think it would be a particularly difficult feature to add to `docu-notion`, but it's just not there.  It's not there for plugins, either.

**Comment by hatton** [comment]
sweet

**Comment by JJJ4032002** [comment]
Hey all, is there any progress regarding this blog automation feature?
